assets/source/setup-guide/app/views/LandingPageApp.js

```diff
@@ -187,9 +190,42 @@ const FaqSection = () => {
 	);
 };
 
-const FaqQuestion = ( { question, answer } ) => {
+/**
+ * Clicking on getting started page faq item to collapse or expand it.
+ *
+ * @event wcadmin_pfw_get_started_faq
+ *
+ * @property {string} action `'expand' | 'collapse'` What action was initiated.
+ * @property {string} question_id Identifier of the clicked question.
+ */
+
+/**
+ * FAQ component.
+ *
+ * @fires wcadmin_pfw_get_started_faq whenever the FAQ is toggled.
+ * @param {Object} props React props
+ * @param {string} props.questionId Question identifier, to be forwarded to the trackign event.
+ * @param {string} props.question Text of the question.
+ * @param {string} props.answer Text of the answer.
+ * @return {JSX.Element} FAQ component.
+ */
+const FaqQuestion = ( { questionId, question, answer } ) => {
+	const panelToggled = useCallback(
+		( isOpened ) => {
+			recordEvent( 'wcadmin_pfw_get_started_faq', {
```

Comment:
❔ 
It shows the event name `wcadmin_wcadmin_pfw_get_started_faq` on DevTool console. Are the repeated prefixes `wcadmin_` expected?

Reply:
Thanks for catching it. :) It's not expected.
I got too attached to FieldGuide's naming convention rule:
> Don’t programmatically create event names. They should be hardcoded so they are easily searchable in the code base.

And `@woocommerce/tracks` already concatentates `wcadmin_`